lib/gollum/views/layout.rb

```diff
@@ -40,6 +40,22 @@ def js # custom js
         @js
       end
 
+      # Passthrough additional omniauth parameters for status bar
+      def user_authed
+        @user_authed
+      end
+
+      def user_provider
+        @user.provider
+      end
+
+      def user_name
+        @user.name
+      end
+
+      def omnigollum_enabled
+        @omnigollum_enabled
```

Comment:
Where is this variable, as well as the ones above, being set? In omnigollum?

Reply:
Yes, they are set in omingollum.